src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,128 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
+import { getMovie } from '../../api/api';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ movies, addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movieFromApi, setMovieFromApi] = useState('');
+  const [error, setError] = useState(false);
+  const [warn, setWarn] = useState(false);
+  const movieToAdd = {
+    title: movieFromApi.Title,
+    description: movieFromApi.Plot,
+    imgUrl: movieFromApi.Poster,
+    imdbUrl: `https://www.imdb.com/title/${movieFromApi.imdbID}/`,
+    imdbId: movieFromApi.imdbID,
+  };
+
+  const addMovieWithBenefits = () => {
+    setQuery('');
+    setMovieFromApi('');
+    addMovie([
+      ...movies,
+      movieToAdd,
+    ]);
+  };
+
+  const addHandler = () => {
+    if (!movies.some(movie => movieFromApi.imdbID === movie.imdbId)
+    && movieFromApi.Response === 'True') {
+      addMovieWithBenefits();
+    } else {
+      setWarn(true);
+      setError(false);
+    }
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={event => event.preventDefault()}
```

Comment:
make request to the api on submit(after user enters the title and hits Enter button)